package/MDAnalysis/analysis/contacts.py

```diff
@@ -143,9 +143,9 @@
     def is_any_closer(r, r0, dist=2.5):
         return np.any(r < dist)
 
-The first two parameters `r` and `r0` are provided by :class:`Contacts` when it
+The first two parameters ``r`` and ``r0`` are provided by :class:`Contacts` when it
```

Comment:
It is in markdown, not in rst. In rst single ticks mean italic. 
Le 22 juil. 2016 6:31 PM, Max Linke <notifications@github.com> a écrit :In package/MDAnalysis/analysis/contacts.py:
> @@ -143,9 +143,9 @@
>      def is_any_closer(r, r0, dist=2.5):
>          return np.any(r < dist)
>  
> -The first two parameters `r` and `r0` are provided by :class:`Contacts` when it
> +The first two parameters ``r`` and ``r0`` are provided by :class:`Contacts` when it

I thought single backticks is enough for the monospace style.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.